### DISCUSSION TITLE
question de débutant

### DISCUSSION THREAD

**Discussion by beautythieves** [twbs/bootstrap#37828] [Q&A]
Bonjour, je découvre bootstrap aujourd'hui car je veux l'utiliser pour me faciliter la tâche pour la création de menus déroulants.
Avant d'installer bootstrap via le cdn, le projet sur lequel je travaille utilise javascript. Or, si j'insère comme indiqué dans la documentation, le lien cdn dans la balise head, certaines fonctionnalités de mon js existant ne marchent plus. est-ce normal?
en vous remerciant.

**Comment by julien-deramond** [comment]
Hey @beautythieves, could you please edit your question so that it is in English?
I'm French so I can already provide an answer.
I'd say that there are probably few chances that Bootstrap JS file causes any issues just by inserting it in but we never know; it'll depend on the content of your project.
In order to be able to understand what would be the cause, we would need a link to your running project or maybe a reproducible example. Otherwise it'll be complicated to provide a technical analysis.

**Comment by beautythieves** [comment]
hello, here is he link of the repository:
https://github.com/beautythieves/Front-End-Fisheye

**Reply by beautythieves** [reply]
hello, 
thank you for your answer and sorry for the dealy.
i have fixed the problem.
